accelerator/config.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
  * @brief Configuration of tangle-accelerator
  */
 
-#define TA_VERSION "tangle-accelerator/0.8.0"
+#define TA_VERSION "tangle-accelerator/0.9.0"
```

Comment:
Why? I supposed `TA_VERSION` was bumped.

Reply:
we accidentally bump the version name on release branch only.